packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -120,6 +121,7 @@ const AccountSwitcher = ({
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
+        RudderStack.setAccountType(loginid.substring(0, 2));
```

Comment:
taken from https://github.com/binary-com/deriv-app/pull/9331/files since we don't know which PR will be merged sooner.